TI-reports/2024/2024-Q4-Sigstore.md

```diff
@@ -0,0 +1,49 @@
+# 2024 Q4 Sigstore
+
+## Overview
+
+### Conference
+
+Our second [SigstoreCon: Supply Chain Day](https://events.linuxfoundation.org/sigstorecon-supply-chain-day/) conference just wrapped up. With just over 90 attendees, SigstoreCon brought together individuals and organizations excited about not only Sigstore but other supply chain initiatives such as SLSA, SBOM, or in-toto. Talks are recorded [here](https://www.youtube.com/playlist?list=PLM6mY5TOhY1E02_fQWqfQk_gMRHHtX0q6).
```

Comment:
Using “here” as a link anchor is an anti-pattern.
For more info see the following blog post (from a renowned expert. ;-) : https://lehors.wordpress.com/2009/01/29/linking-the-proper-way/